sympy/functions/elementary/miscellaneous.py

```diff
@@ -60,7 +60,7 @@ def __new__(cls):
 ###############################################################################
 
 
-def sqrt(arg):
+def sqrt(arg, evaluate=True):
```

Comment:
After looking at the source, I noticed the default value of evaluate is None (as seen [here](https://github.com/sympy/sympy/blob/master/sympy/core/power.py#L245), so it should be None instead of True. 
When None, the evaluation is governed by global flag `global_evaluate` which is typically True but need not always be.